### PR TITLE
daily pr from master to published

### PR DESCRIPTION
**Pull request by DawnWood-Docker**

### CODE REVIEW

File: app-template/working-with-template.md [hunk 1]
```diff
@@ -10,30 +10,47 @@ keywords: Docker, application template, Application Designer,
...
+templates. To create a custom template based on your requirements, you must
+complete the following steps:
 
 1. Create a service container image
 2. Create the service template definition
 3. Add the service template to the library
 4. Share the service template
 
-### Create a service container image
+### A service container image
```
Comment (lena-larionova) [inline app-template/working-with-template.md]: Not sure why this was changed, the previous titled seems to make more sense, and is more consistent with the other sections.

File: app-template/working-with-template.md [hunk 2]
```diff
@@ -10,30 +10,47 @@ keywords: Docker, application template, Application Designer,
...
 
-A service template provides the description required by Docker Template to scaffold a project. A service template runs inside a container with two bind mounts:
+A service template provides the description required by Docker Template to
+scaffold a project. A service template runs inside a container with two bind
+mounts:
 
-1. `/run/configuration`, a JSON file which contains all settings such as parameters, image name, etc. For example:
+1. `/run/configuration`, a JSON file which contains all settings such as
```
Comment (lena-larionova) [inline app-template/working-with-template.md]: If you look at this in the preview, there are now two "step 1"s in this section.

**Comment by DawnWood-Docker** [comment]
closing so that we can fix the issues that Lena raises. We'll republish once those are fixed.

File: app-template/working-with-template.md (hunk 2 above)
Comment (usha-mandya) [reply app-template/working-with-template.md]: Thanks for spotting this, @lena-larionova. This wasn't displayed as expected due to the code between the numbered list. This is fixed in https://github.com/docker/docker.github.io/pull/9537.

File: app-template/working-with-template.md (hunk 1 above)
Comment (usha-mandya) [reply app-template/working-with-template.md]: Yup, makes sense. The title has been updated in https://github.com/docker/docker.github.io/pull/9537.

**Pull request closed without merging**